tests/vmi_multus_test.go

```diff
@@ -65,6 +67,16 @@ var _ = Describe("Multus", func() {
 	virtClient, err := kubecli.GetKubevirtClient()
 	tests.PanicOnError(err)
 
+	useExistingCluster := true
```

Comment:
please consider move this to https://github.com/kubevirt/kubevirt/pull/3551/commits/b02341089320843590a7ea40b968bcccd9926ae8
maybe it is more related to that one

Reply:
I addressed this in a new commit